### ISSUE TITLE
TWL-JOGL port

### ISSUE THREAD

**Issue by maheshkurmi** [MatthiasMann/twl#3]
I have written [jogl port of twl]( https://github.com/maheshkurmi/jogl-twl) and it is working like charm.
Thanks for sharing your valuable work with us.
BTW I had to change keycodes and mouse button-id codes along with new renderers to get it working.
You can share jogl-port of twl wherever it seems suitable.

**Comment by MatthiasMann** [comment]
Nice :)

**Comment by maheshkurmi** [comment]
I think there must be default theme already loaded in TWL, since I think learning curve to theme widgets is steep, moreover it takes significant time in loading theme (lot of xml). For me the loading time for complete _sample_demo.xml_ theme is approx 1.3 seconds.
For newbies TWL seems to be lot difficult because of its powerful theme engine.
What about default vector rendered theme (widgets are drawn using geometry only)?

**Comment by MatthiasMann** [comment]
No, the core of the library is the separation of rendering from the widgets.

I have used various different themes for games and other apps. In most 
of these you can't even see that a widget library is being used.

The parsing speed is dependent on the complexity of the theme and the 
amount / size of images. Did you do profiling to see if the XML is 
actually the bottle neck? because the XPP library is really fast.
